kedro_azureml/generator.py

```diff
@@ -172,12 +172,22 @@ def _get_output(self, name):
         if name in self.catalog.list() and isinstance(
             ds := self.catalog._get_dataset(name), AzureMLAssetDataSet
         ):
+            output_path = (
+                f"azureml://datastores/{ds._datastore}/paths/{ds._azureml_root_dir}"
```

Comment:
We probably need to strip off trailing slashes from `ds._azureml_root_dir`, or use some URI parsing lib